main.py

```diff
@@ -51,12 +52,12 @@
 
     # generate requested market
     if "pac" in cfg.market_type:
-        m = market_2pac.TwoSidedPayAsClear(0)
+        m = market_2pac.TwoSidedPayAsClear(0, network=sc.power_network)
```

Comment:
Includes clusters to orders, but without influence on the prices as this is done during matching.
- This should be tackled within another PR